### ISSUE TITLE
Literally 50% Win Rate NOT Statistically

### ISSUE THREAD

**Issue by cedricchi** [ValveSoftware/Dota2-Gameplay#16304]
### Description

11 games losing streak followed by 11 games win streak (steam ID 108901726)
I think for Valve "balance" means keeping you hooked to the game for eternity by forcing 50% winrate. When you have win streak you will be given the worst possible player worse than your imagination. These bots will feed, troll, jungle to make sure you will lose.
Valve will never make their matchmaking algorithm public, because of the dark nature of their business strategy.
On the other hand, people are not as foolish as @jeffhill, they simply leave dota 2 once they see the ridiculous quality of game experience.
The ones that are left use unfair services such as Overplus, boosting, account selling on Taobao, truly toxic people.
Recently they added the feature that allow Dota+ player to deny a low quality match only to be given even worse team. I personally experienced 11-games losing streak!!! I want to vomit
By the way, the skill differences of our games are so huge you can compare them to heaven and earth. Stomps make me want to kill myself.

### Example Match ID (and possibly Timestamp)

steam ID 108901726

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/57655363/abf7605f-3b25-4346-a200-0a768ef65984)

**Comment by TobyFlendersonFromHR** [comment]
1490 mmr, laughable tbh

**Comment by cedricchi** [comment]
> 1490 mmr, laughable tbh

Alright @TobyFlendersonFromHR ,
I played VS platform Dota tournament and won prize money only to be laughed by a second class HR?
You think your ranked system measures something?
The thing that really deserves laughing is your team and your education background. I did some digging on your pathetic degrees, if you would like to call them degree.

**Comment by IIaBeJl96** [comment]
Yes, you're completely right, this is a 4ckin9 casino and nothing more. I calibrated Archon 2 and within a week I upgraded myself to Archon 3, and as soon as I upgraded Archon 3 the real nightmare began, I fell already lower than I had calibrated. For me he throws Furionov 5, Bounty Hunter 4, Crusader carry, Windranger ruiner 4, who said at 1 minute that the fault of my win streak was myself, then I combined this game, and he got afk in the forest!

**Comment by IIaBeJl96** [comment]
> Да, вы совершенно правы, это казино 4ckin9 и не более того. Я откалибровал Архонт 2 и через неделю обновился до Архонта 3, а как только обновил Архонт 3 начался настоящий кошмар, я упал уже ниже откалиброванного. Мне он кидает Фурионова 5, Баунти Хантера 4, Крестоносца керри, Виндрейнджера руинера 4, который на 1 минуте сказал, что в моей победной серии виноват я сам, потом я объединил эту игру, и он получил афк в лесу!


Immediately after calibration W-L: 2-2; 4-5; 5-4; 5-9 (FRIDAY, BEER GAMERS); 4-5; 1-1 (2 games per day, because I'm already tired and hate the selection of this crap); 2-3 (Today, loose, 2 wins, and 2 more looses, I predict next wins)

**Comment by cedricchi** [comment]
> Yes, you're completely right, this is a 4ckin9 casino and nothing more. I calibrated Archon 2 and within a 3 days I upgraded myself to Archon 3, and as soon as I upgraded Archon 3 the real nightmare began, I fell already lower than I had calibrated. For me he throws Furionov 5, Bounty Hunter 4, Crusader carry, Windranger ruiner 4, who said at 1 minute that the fault of my loose streak was myself, then I comeback this game, and he got afk in the forest!
> 
> Immediately after calibration Win-Loose: 2-2; 4-5; 5-4; 5-9 (FRIDAY, BEER GAMERS); 4-5; 1-1 (2 games per day, because I'm already tired and hate the selection of this crap); 2-3 (Today, loose, 2 wins, and 2 more looses, I predict next wins)

I feel you

**Comment by TobyFlendersonFromHR** [comment]
> > 1490 mmr, laughable tbh
> 
> Alright @TobyFlendersonFromHR , I played VS platform Dota tournament and won prize money only to be laughed by a second class HR? You think your ranked system measures something? The thing that really deserves laughing is your team and your education background. I did some digging on your pathetic degrees, if you would like to call them degree.

you won $15 dollars at your local net cafe with a tournament of only 1k mmr players, don't @ me

**Comment by cedricchi** [comment]
> > > 1490 mmr, laughable tbh
> > 
> > 
> > Alright @TobyFlendersonFromHR , I played VS platform Dota tournament and won prize money only to be laughed by a second class HR? You think your ranked system measures something? The thing that really deserves laughing is your team and your education background. I did some digging on your pathetic degrees, if you would like to call them degree.
> 
> you won $15 dollars at your local net cafe with a tournament of only 1k mmr players, don't @ me

Okay, again this is harassment. I am going to block you, good luck with your orphan life.
